src/main/resources/org/sagebionetworks/web/client/view/EmailInvitationViewImpl.ui.xml

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+			 xmlns:g="urn:import:com.google.gwt.user.client.ui"
+			 xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+			 xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
+>
+
+	<g:HTMLPanel>
+		<b:Row>
+			<b:Column size="XS_12, MD_6" offset="MD_3">
+				<g:SimplePanel ui:field="synapseAlertContainer"/>
+			</b:Column>
+		</b:Row>
+		<div class="container margin-top-15 min-height-400">
```

Comment:
remove div, add class names to Container below instead (addStyleName="margin-top-15 min-height-400")